### DISCUSSION TITLE
Equivalent de l'acteur Redy "Compteur Journal"

### DISCUSSION THREAD

**Discussion by edejean** [witsa/synapps#520] [Q&A] [answered]
Bonjour,
Tout est dans le titre ;)
Quel serait l'équivalent de l'acteur Redy "Compteur Journal" ?
Cdt
Eric

**Comment by WITNicolas** [comment]
Bonjour,
Malheureusement (à cet instant) il n'existe pas un tel acteur.
Par contre il est possible de "contourner" légèrement ce manque.

Il faut :
Ajouter l'acteur Journal. lui donner une taille de 0px / 0px (ne pas le rendre invisible, sinon l'acteur ne requête plus)
Ajouter un acteur Texte qui sera liaisonné à l'acteur Journal et en utilisant la propriété Nombre située dans information.

**Comment by edejean** [comment]
Bonjour,
Et si je veux compter seulement les alarmes concernant un équipement en particulier et celles qui ne sont pas acquittés, la partie script n'étant pas documentée, une solution ?
Cdt
Eric

**Reply by edejean** [reply]
J'ai créé une autre source de données pointant sur un ancien synoptique utilisant les acteurs "Compteur Journal" et je pointe sur le chemin "total.Param.EventCount" afin de récupérer la valeur qui m'intéresse...en attendant mieux...

**Reply by manufitoussiwit** [reply]
Oui effectivement, Il va falloir ajouter la possibilité de filtrer le journal par statut d'acquittement.

**Comment by WITNicolas** [comment, marked as answer]
Dans la version 1.1.6 il est maintenant possible de filtrer selon plusieurs criteres et d'en retirer le nombre d'evenements ainsi filtrés
